content/src/cloud/features/cloud-backup-restoring-from.md

```diff
@@ -6,3 +6,18 @@ zendesk:
   labels: cloud
 ---
 
+## Prerequisites
+
+- You have administrator rights in Home Assistant
+- You have [enabled Home Assistant Cloud backups](/hc/en-us/articles/26294320337181/)
+- You have [downloaded the backup](/hc/en-us/articles/26294370569245/) you want to restore from
```

Comment:
As you can now restore during onboarding, these 3 are no longer required, right?